TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Shared/_Layout.cshtml

```diff
@@ -1,70 +1,57 @@
+                        </svg>
+                        <!--<![endif]-->
+                        <!--[if IE 8]>
+                        <img src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image" width="36" height="32">
+                        <![endif]-->
+                        <span class="govuk-header__logotype-text">
+                            GOV.UK
+                        </span>
+                    </span>
+                </a>
+            </div>
+            <div class="govuk-header__content">
+                <a asp-area="" asp-page="/Index" class="govuk-header__link govuk-header__link--service-name">
+                    Teaching Record System
+                </a>
+                @RenderSection("HeaderNav", required: false)
             </div>
-        </nav>
+        </div>
     </header>
-    <div class="container">
-        <main role="main" class="pb-3">
-            @RenderBody()
-        </main>
-    </div>
+}
 
-    <footer class="border-top footer text-muted">
-        <div class="container">
-            &copy; 2023 - TeachingRecordSystem.SupportUi - <a asp-area="" asp-page="/Privacy">Privacy</a>
-        </div>
-    </footer>
+@RenderBody()
 
-    <script src="~/lib/jquery/dist/jquery.min.js"></script>
-    <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-    <script src="~/js/site.js" asp-append-version="true"></script>
+@section Footer {
+    <footer class="govuk-footer " role="contentinfo">
+        <div class="govuk-width-container ">
+            <div class="govuk-footer__meta">
+                <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
 
-    @await RenderSectionAsync("Scripts", required: false)
-</body>
-</html>
\ No newline at end of file
+                    <svg aria-hidden="true" focusable="false" class="govuk-footer__licence-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 483.2 195.7" height="17" width="41">
+                        <path fill="currentColor" d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145" />
+                    </svg>
+                    <span class="govuk-footer__licence-description">
```

Comment:
Don't want this block - our content is available under OGL

Reply:
removed but kept div for the moment to keep the Crown logo on the right